scalatest/src/main/scala/org/scalatest/Shell.scala

```diff
@@ -377,7 +377,7 @@ sealed trait Shell {
    * this method is invoked.
    * </p>
    */
-  def run(suite: Suite, testName: String = null, configMap: ConfigMap = ConfigMap.empty): Unit
+  def run(suite: Suite, testName: String | Null = null, configMap: ConfigMap = ConfigMap.empty): Unit
```

Comment:
**Nullable parameter**
Parameter that can be null and should therefore have a `| Null` type.